src/java.xml/share/classes/module-info.java

```diff
@@ -221,6 +220,85 @@
  * <tbody>
  *
  * <tr>
+ * <th scope="row" style="font-weight:normal" id="EELimit">entityExpansionLimit</th>
+ * <td>Limits the number of entity expansions.
+ * </td>
+ * <td rowspan="9">yes</td>
+ * <td rowspan="9">yes</td>
+ * <td rowspan="9">Integer</td>
+ * <th id="Value" scope="row" style="font-weight:normal" rowspan="9">
+ * A positive integer. A value less than or equal to 0 indicates no limit.
+ * If the value is not an integer, a NumericFormatException is thrown.
+ * </th>
+ * <th id="Default" scope="row" style="font-weight:normal">64000</th>
```

Comment:
There are other ```Default``` ids in other rows. Should they be distinct? Or rather needed?

Reply:
True, not needed. Will remove, and also look at other copy&paste-d ids.